File: training-data/java/BinaryDecimalConvert.java
public class BinaryDecimalConvert {
    public static int binaryToDecimal(String binary) {
        return Integer.parseInt(binary, 2);
    }

    public static String decimalToBinary(int n) {
        return Integer.toBinaryString(n);
    }

    public static void main(String[] args) {
        System.out.println(binaryToDecimal("1010"));
        System.out.println(decimalToBinary(10));
    }
}

File: training-data/javascript/binary_decimal_convert.js
function binaryToDecimal(binary) {
  return parseInt(binary, 2);
}

function decimalToBinary(n) {
  return n.toString(2);
}

console.log(binaryToDecimal('1010'));
console.log(decimalToBinary(10));

module.exports = { binaryToDecimal, decimalToBinary };
